kddnewton/regular_expression
# General cleanup

* Rename `web-demo` to `demo`
* Add `demo` test to the GitHub actions workflow
* Move tests around for consistency
* Remove some unnecessary calls in `parser.y`
* Convert demo tests to minitest for consistency
* Make sure demo passes rubocop
